File: Python/Excel Sheet Column Title.py
class Solution:
    # @return a string
    def convertToTitle(self, num):
        s = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
        st = ''
        while num > 0:
            cnt = num % 26
            st += s[(cnt + 25) % 26]
            num = num/26
            if cnt == 0:
                num -= 1
        return st[::-1]

File: C++/Excel-Sheet-Column-Title.cpp
class Solution {
public:
    string convertToTitle(int n) {
    	string res;
    	while (n != 0) {
    		res += ('A' + (n - 1) % 26);
    		n = (n - 1) / 26;
    	}
    	reverse(res.begin(), res.end());
    	return res;
    }
};